packages/react-devtools-core/src/standalone.js

```diff
@@ -44,6 +41,12 @@ function reload() {
   ReactDOM.unmountComponentAtNode(node);
   node.innerHTML = '';
   setTimeout(() => {
+    if (typeof window.__REACT_DEVTOOLS_GLOBAL_HOOK__ === 'undefined') {
+      window.__REACT_DEVTOOLS_GLOBAL_HOOK__ = {};
+    }
+    window.__REACT_DEVTOOLS_GLOBAL_HOOK__.launchEditor = (fileName, lineNumber) => {
```

Comment:
I wasn't sure if it'd ever get overwritten and we wanted to make sure it was reinstalled; if that's not a concern I'll simply hoist it.